content/en/docs/Reference/Releases/regolith-1.5.3-release-notes.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "1.5.3 Release Notes"
+linkTitle: "1.5.3 Release Notes"
+weight: 1
+description: >
+  Release notes for Regolith 1.5.3.
```

Comment:
Smallest of nitpicks: can you remove the dot at the end of the line? The description gets translated into a h3 or h4, which looks weird ending in a dot

Reply:
Yep, fixed thanks :)